docs/_interface/Managing-your-accounts.md

```diff
@@ -52,301 +26,233 @@ COMMANDS:
-you are used to may change. Therefore you make sure you either do not rely on the account
-order or doublecheck and update the indexes used in your scripts.
+```shell
+geth account update a94f5374fce5edbc8e2a8697c15331677e6ebf0b --password path/password.txt
+```
 
-**Warning:** If you use the password flag with a password file, best to make sure the file
-is not readable or even listable for anyone but you. You achieve this with:
+Updating the account replaces the original file with a new one - this means the original file is no longer available after it has been updated.
 
-```
-touch /path/to/password 
-chmod 700 /path/to/password
-cat > /path/to/password
->I type my pass here^D
-```
 
-## Updating an existing account
+## Unlocking accounts
 
-You can update an existing account on the command line with the `update` subcommand with
-the account address or index as parameter. You can specify multiple accounts at once.
+In Geth, accounts are locked unless they are explicitly unlocked. If an account is intended to be used by apps connecting to Geth via RPC then it can be unlocked in non-interactive mode by passing the `--unlock` flag with a comma-separated list of account addresses (or keystore indexes) to unlock. This unlocks the accounts for one session only. Including the `--unlock` flag without any account addresses defaults to unlocking the first account in the keystore. 
 
-```
-geth account update 5afdd78bdacb56ab1dad28741ea2a0e47fe41331 9acb9ff906641a434803efb474c96a837756287f
-geth account update 0 1 2
+```shell
+geth <other commands> --unlock 0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b
 ```
 
-The account is saved in the newest version in encrypted format, you are prompted
-for a passphrase to unlock the account and another to save the updated file.
+Geth will start and prompt the user to input the account password in the terminal. Alternatively, the user can provide a password as a text file and pass its path to `--password`:
 
-This same command can therefore be used to migrate an account of a deprecated
-format to the newest format or change the password for an account.
+```shell
+geth <other commands> --unlock 0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b --password path/password.txt
+```
 
-After a successful update, all previous formats/versions of that same key are removed!
+{% include note.html content=" By default, account **unlocking is forbidden when HTTP or Websocket access is enabled** (i.e. by passing `--http` or `ws` flag). This is because an attacker that manages to access the node via the externally-exposed HTTP/WS port can then control the unlocked account. It is possible to force account unlock by including the `--allow-insecure-unlock` flag but this is not recommended if there is any chance of the node connecting to Ethereum Mainnet. This is not a hypothetical risk: **there are bots that continually scan for http-enabled Ethereum nodes to attack**" %}
```

Comment:
> It is possible to force account unlock by including the `--allow-insecure-unlock` flag but this is not recommended if there is any chance of the node connecting to Ethereum Mainnet.

This has nothing to do with connecting to Ethereum Mainnet. Rather, "if there is any chance of an attacker interacting with the RPC interface". 

But I'd rather say something which expresses that it's unsafe and not recommended unless you are enough of an expert to know exactly in what circumstances it is safe to do so.